pkg/volume/plugins.go

```diff
@@ -58,8 +59,8 @@ type Plugin interface {
 	NewCleaner(name string, podUID types.UID) (Cleaner, error)
 }
 
-// Host is an interface that plugins can use to access the kubelet.
-type Host interface {
+// NodeHost is an interface that plugins can use to access the kubelet.
+type NodeHost interface {
```

Comment:
"Host" is pretty well established as "the thing that runs plugins" in a number of plugin-centric software systems.  It's not a manager - it's an interface back to the app that hosts the plugin, but in this case it is the app that hosts NodePlugins.

I sort of hate the naming.  We could play tricks with sub-packages or we could just go back to the old names for Node* and just make the soon-to-exist Persistent* be decorated names.